dev-lang/gnucobol/gnucobol-2.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="an open-source COBOL compiler"
+HOMEPAGE="https://open-cobol.sourceforge.io"
+SRC_URI="mirror://sourceforge/open-cobol/${P}.tar.gz"
+
+LICENSE="GPL-3+ LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="berkdb nls"
```

Comment:
We may need to pull in libntl or gettext if nls is enabled?

Reply:
Added gettext - looks it is used by the source code.

Reply:
In the past, I have done:
BDEPEND="sys-devel/gettext"
DEPEND="virtual/libintl" (to allow for Musl where we need to pull in a package)

But I had to do some investigation to figure out what was needed and when. If you can't figure it out for now, we will leave it.